### DISCUSSION TITLE
Disable infinite scroll on desktop carousel

### DISCUSSION THREAD

**Discussion by galleta5** [elementor/elementor#24118] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I want to display 3 slides on my carousel and would like to turn off the 'infinite scroll' on desktop without having to turn it off on mobile where I do need it on. Also to have 'offset sides' not be dependant on whether or not 'infinite scroll' is turned on.

### Describe the solution you'd like

Option to have infinite scroll on or off for each mode (desktop, tablet, mobile) 

### Describe alternatives you've considered

Make 1 carousel for desktop and another for mobile.

**Comment by edmis888** [comment]
Did you find a solution? I want to implement exact same thing.
